frontend/src/components/Navbar.js

```diff
@@ -46,6 +53,32 @@ function CourseTableNavbar({ isLoggedIn }) {
     window.location.pathname = '/';
   };
 
+  // DarkMode or LightMode
+  setFetchMethod(window.fetch);
```

Comment:
what does this do?

Reply:
not 100% sure on all the details but sometimes the darkreader library needs to access "a cross-origin resource" and without this line it throws an error